bgzip_utils/bgzip_utils.pyx

```
@@ -209,7 +228,15 @@ def decompress_into(bytes src_buff, bytearray dst_buff, int num_threads):
             out += blocks[i].inflated_size
 
         for i in prange(number_of_blocks, num_threads=num_threads, schedule="dynamic"):
-            inflate_block(&blocks[i])
+            err = inflate_block(&blocks[i])
+            if BGZIP_OK == err:
+                pass
+            elif err == BGZIP_ZLIB_ERROR:
+                with gil:
+                    raise BGZIPException("zlib error encountered during inflation")
+            else:
+                with gil:
+                    raise BGZIPException()
```

Should this have a message?

---

This is a catch-all exception for now. I'll add a TODO to improve messaging.